misc/app.ts

```diff
@@ -0,0 +1,6 @@
+import { isSsr } from "@/constants";
+import createApp from "@saleor/app-bridge";
+
+export const app = !isSsr ? createApp() : undefined;
```

Comment:
Hmm, maybe it should be wrapped in `useMemo`?

Reply:
It is intended in order to get the latest token, similarly like in the checkout app.